courses/views/v1/__init__.py

```diff
@@ -26,14 +26,17 @@
     Program,
     CourseRunEnrollment,
     ProgramEnrollment,
+    ProgramCertificate,
 )
 from courses.serializers import (
     CourseRunEnrollmentSerializer,
     CourseRunSerializer,
     CourseSerializer,
     ProgramSerializer,
     UserProgramEnrollmentDetailSerializer,
+    ProgramCertificateSerializer,
```

Comment:
Not sure if we are using this or the above import?

Reply:
This was being used initially but then I moved it to another serializer and forgot to remove it. Now removed it